### ISSUE TITLE
Issue with URL

### ISSUE THREAD

**Issue by NedFlanders2** [hackademix/noscript#197]
Having an issue with noscript extension and a certain website. Basically the site will load if i set to temp trusted but the page will not work properly when clicking on links. The address will change in the address bar when i click on links but will not actually update the page until i refresh the website manually.

_Tried the following_:
- Temp trusted website > updates initial pages as expect but then clicking on any following links results in issues as stated above.
- Disable noscript extension completely > same issue/problem
- Removed noscript from the browser >url works as intended

Inspected noscript and found the following not sure if this is of any help but it is all i could find when digging around and contains said url:

**Comment by NedFlanders2** [comment]
TypeError: can't access dead object webconsole.js:272:9
sendRemoveListener on closed conduit {73a6fe31-595d-460b-a920-fcc0f8843232}.274877908785 ConduitsChild.jsm:108
[NoScript]:0 deferWebTraffic on  
Promise { <state>: "fulfilled", <value>: undefined }
log.js:13:13
TypeError: can't access dead object webconsole.js:272:9
[NoScript]:1 Including /nscl/service/prefetchCSSResources.js log.js:13:13
TypeError: can't access dead object webconsole.js:272:9
[NoScript]:2 Including /legacy/Legacy.js log.js:13:13
TypeError: can't access dead object webconsole.js:272:9
[NoScript]:3 Including /xss/Exceptions.js log.js:13:13
TypeError: can't access dead object webconsole.js:272:9
[NoScript]:4 Including /legacy/defaults.js log.js:13:13
TypeError: can't access dead object webconsole.js:272:9
[NoScript]:5 Including /nscl/common/restricted.js log.js:13:13
TypeError: can't access dead object webconsole.js:272:9
[NoScript]:6 Collected seen 
Array(4) [ {…}, {…}, {…}, {…} ]
log.js:13:13
[NoScript]:7 Recording 
Object { key: "noscript-probe", url: "https://manual.yoyogames.com/#t=IDE_Navigation%2FBookmarks.htm", documentUrl: "https://manual.yoyogames.com/#t=IDE_Navigation%2FBookmarks.htm", type: "main_frame", tabId: 1, frameId: 0 }
log.js:13:13
TypeError: can't access dead object webconsole.js:272:9
TypeError: can't access dead object webconsole.js:272:9
[NoScript]:8 Recording 
Object { key: "script@https://manual.yoyogames.com<https://manual.yoyogames.com", type: "script", url: "https://manual.yoyogames.com/template/scripts/rh.min.js", documentUrl: "https://manual.yoyogames.com/#t=IDE_Navigation%2FBookmarks.htm", originUrl: "https://manual.yoyogames.com/#t=IDE_Navigation%2FBookmarks.htm", initialUrl: "https://manual.yoyogames.com/template/scripts/rh.min.js", tabId: 1 }
log.js:13:13
[NoScript]:9 Recording 
Object { key: "sub_frame@https://manual.yoyogames.com<https://manual.yoyogames.com", type: "sub_frame", url: "https://manual.yoyogames.com/Content.htm", documentUrl: "https://manual.yoyogames.com/#t=IDE_Navigation%2FBookmarks.htm", originUrl: "https://manual.yoyogames.com/", initialUrl: "https://manual.yoyogames.com/Content.htm", tabId: 1 }
log.js:13:13
[NoScript]:10 Recording 
Object { key: "main_frame@https://manual.yoyogames.com<https://manual.yoyogames.com", type: "main_frame", url: "https://manual.yoyogames.com/#t=IDE_Navigation%2FBookmarks.htm", documentUrl: "https://manual.yoyogames.com/#t=IDE_Navigation%2FBookmarks.htm", originUrl: undefined, tabId: 1 }
log.js:13:13
TypeError: can't access dead object 3 webconsole.js:272:9
[NoScript] STARTED log.js:9:13
[NoScript]:11 Including /bg/popupHandler.js log.js:13:13
TypeError: can't access dead object webconsole.js:272:9
TypeError: can't access dead object webconsole.js:272:9
Unchecked lastError value: Error: Missing host permission for the tab 2 background.js:18554

**Comment by hackademix** [comment]
Hello, those messages in the browser console are very likely unrelated.
Those in the pages' console (ctrl+shift+K) may be more useful.
Even more useful would be knowing the exact site where your problem happen and your complete NoScript configuration (NoScript Options>Export).
Feel free to share privately [by email](https://maone.net), if it's sensitive.

**Comment by NedFlanders2** [comment]
ok ctrl+shift+k

When ever i click on a link within the page it gives this error 
Uncaught DOMException: An exception was thrown 4b8b2aa7-0cb5-4b9f-a55b-5913c3f77b55:59
each time i click something that error will regenerate a new one.

url: https://manual.yoyogames.com/#t=Content.htm

I know it is to do with noscript because i have eliminated all other extensions one by one and even no extensions installed at all. It is only when i remove noscript completely that the website will work as intended.

I contacted the devs of the website too before i came here so they know about it also. Wether it is on noscripts end or their end i don't know but all i know is that i need this to work because i have used noscript for years and don't want to stop for one website. Noscript is a staple for me and i don't browse with out it.

This url in question is a manual i need to reference constantly so its obtrusive and slow to use atm because of the issue.   

I have used noscript for a long time and never have i once seen a situation where i needed to uninstall the extension completely for a website to work. The website will not even work if noscript is disabled. Seems pretty strange to me because in theory the website should work if noscript is disabled.
